docs/src/pages/components/menus/DenseMenu.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import Paper from '@material-ui/core/Paper';
+import Divider from '@material-ui/core/Divider';
+import MenuList from '@material-ui/core/MenuList';
+import MenuItem from '@material-ui/core/MenuItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+
+import Check from '@material-ui/icons/Check';
+import ArrowRight from '@material-ui/icons/ArrowRight';
+
+export default function DenseMenu() {
+  return (
+    <Paper sx={{ width: 320, maxWidth: '100%' }}>
+      <MenuList dense>
+        <MenuItem>
+          <ListItemText inset>Single</ListItemText>
+        </MenuItem>
+        <MenuItem>
+          <ListItemText inset>1.15</ListItemText>
+        </MenuItem>
+        <MenuItem>
+          <ListItemText inset>Double</ListItemText>
+        </MenuItem>
+        <MenuItem>
+          <ListItemIcon>
+            <Check />
+          </ListItemIcon>
+          Custom: 1.2
+        </MenuItem>
+        <Divider />
+        <MenuItem>
+          <ListItemText>Add space before paragraph</ListItemText>
+        </MenuItem>
+        <MenuItem>
+          <ListItemText>Add space after paragraph</ListItemText>
+          <ArrowRight color="action" sx={{ mr: -1 }} />
```

Comment:
It might be a bit misleading considering we don't yet have nested menu support #20591.
The icon doesn't work in RTL, which makes me think about #22726.

Reply:
Okay, I am fine to remove it.